### ISSUE TITLE
does LayoutLM require bboxes & text for token classification?

### ISSUE THREAD

**Issue by parulsingh23** [microsoft/unilm#696]
Can LayoutLM function as an object detection model for documents? 
I need to find & label portions of a document into 'question' and 'answer' regions, and fine-tuning on detectron2 had very poor results. I tried looking for a model pretrained with documents images & doing 'dense' object detection, and the only model I could find was LayoutLM. 

However, I see in documentation that LayoutLM will use the bouding boxes of text & the OCR'ed text as input, which sort of defeats the purpose of 'object detection' here since my purpose for the model is to find the bounding boxes & label them.

Is my understanding of LayoutLM correct, and do you have any advice on dense object detection models for documents?

**Comment by wolfshow** [comment]
@parulsingh23 Yes, LayoutLM needs OCR'ed text as input, because natural language provides more hints for detecting "question" and "answer" pairs. For bbox output, you may need to use any off-the-shelf OCR tools or train your own models with our pre-trained DiT (https://github.com/microsoft/unilm/tree/master/dit).
